model_demos/cv_demos/yolo_v5/pytorch_yolov5_480.py

```diff
@@ -66,6 +66,12 @@ def run_pytorch_yolov5_480(variant="yolov5s"):
                 os.environ["PYBUDA_INSERT_SLICE_FOR_CONCAT"] = "1"
                 os.environ["PYBUDA_CONCAT_SLICE_Y"] = "10"
                 os.environ["PYBUDA_FORCE_CONV_MULTI_OP_FRACTURE"] = "1"
+                os.environ["PYBUDA_MAX_FORK_JOIN_BUF"] = "1"
```

Comment:
Did you tested out if this change breaks GS E150 and GS E300?

Reply:
It will not break on GS E150 and GS E300 because there is condition for checking devices in yolov5 model script

`if available_devices[0] == BackendDevice.Grayskull:`

Reply:
Great, missed that one! :))